static/app/views/performance/landing/widgets/widgets/histogramWidget.tsx

```diff
@@ -80,6 +80,7 @@ export function HistogramWidget(props: Props) {
               field={props.fields[0]}
               chartData={provided.widgetData.chart?.data?.[props.fields[0]]}
               disableXAxis
+              disableZoom
```

Comment:
We've never had an un-zoom feature, is there a reason why it's a necessity now?

Reply:
You can zoom in a bunch of the widgets but there is no way to reset and unzoom, we're also currently not making 2nd requests to get the zoomed in data, I'd rather just exclude the ability for now.